styles/style.css

```diff
@@ -0,0 +1,235 @@
+/* Reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* General */
+.links-style {
+  text-decoration: none;
+}
+
+.blue {
+  color: #08c;
+}
+
+.change-button:hover {
+  cursor: pointer;
+  background-color: rgb(0, 0, 0, 0.3);
+}
+
+.underline-text:hover {
+  text-decoration: underline;
+}
+
+/* Body */
+body {
+  font-family: 'Quicksand', sans-serif;
+}
+
+.container {
+  max-width: 1436px;
+  margin: 0 auto;
+}
+
+/* Navigation Bar */
+.first-container {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  max-width: 100%;
+  background-image: url('https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg');
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  background-position: bottom;
+}
+
+.global-header {
+  background:
+    -webkit-gradient(
+      linear,
+      0 0,
+      0 100%,
+      from(rgba(0, 0, 0, 0.2)),
+      color-stop(0.05, rgba(0, 0, 0, 0)),
+      color-stop(0.97, rgba(0, 0, 0, 0)),
+      to(rgba(0, 0, 0, 0.45))
+    ),
+    -webkit-gradient(
+      linear,
+      0 0,
+      100% 0,
+      from(rgba(0, 0, 0, 0.2)),
+      color-stop(0.002, rgba(0, 0, 0, 0)),
+      color-stop(0.998, rgba(0, 0, 0, 0)),
+      to(rgba(0, 0, 0, 0.2))
+    ),
+    -webkit-gradient(
+      linear,
+      0 100%,
+      0 0,
+      from(#666),
+      color-stop(0.5, #5e5e5e),
+      color-stop(0.51, #707070),
+      to(#808080)
+    );
+  display: flex;
+  flex-flow: row nowrap;
+  width: 980px;
+  height: 36px;
+  border-radius: 6px;
+  box-shadow: rgb(0, 0, 0, 0.3) 0 1px 2px;
+  margin-top: 40px;
+  position: relative;
+  top: 46px;
+}
+
+.nav-bar {
+  display: flex;
+  flex-flow: row nowrap;
+  width: 100%;
```

Comment:
By default is 100%.
```suggestion
```

Reply:
I've erased it, thanks!

Reply:
When I erase this property it messes everything on my navbar so this I had to leave it there.

Reply:
Is not the navbar width the property that you want to change, use `flex-basis: 100%;` (or `flex-basis: fill;`) with the navbar, because it is a flex item.